src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/CalculationResults.kt

```diff
@@ -0,0 +1,8 @@
+package uk.gov.justice.digital.hmpps.calculatereleasedatesapi.model
+
+import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.validation.ValidationMessage
+
+data class CalculationResults(
```

Comment:
This is currently only used by the 1000 calcs, but the plan is to use it for the calulation route as well; i.e. instead of having a separate flow for validation, the validation runs whenever a calculation is requested and messages are returned if it failed.